web/source/osk/oskLayer.ts

```diff
@@ -76,10 +76,10 @@ namespace com.keyman.osk {
      *  @param    {string}  keyId   key identifier
      *  @return   {Object}          Reference to key
      */
-    public findKey(coreID: string): OSKBaseKey {
+    private findKey(keyId: string): OSKBaseKey {
```

Comment:
Curious as to why you are making this `private` now? Not sure how that impacts the usage of this function.

Have you audited all usages of `findKey` to verify that the change of parameter won't break them?

Reply:
The only uses are in this file, almost immediately before the function's definition, so `private` is safe.

Maybe I could make a separate version for the two ID types, if that's a concern?  Though I'd need to convert the other calls in the class to _also_ use the new type, as they're technically subject to the same type of bug that caused the blank spacebar in this context.

Reply:
@mcdurdin Have a preference here?  Should I go ahead and merge as-is?

Reply:
Go ahead and merge as-is I think.